generate-certs/outputs.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
a stub, not sure what we can get out of the box. I'd like to steal some stuff out of systemd of possible?